testing/databaseTest.py

```diff
@@ -116,6 +116,14 @@ def test_kinetics(self):
             self.compat_func_name = test_name
             yield test, family_name
 
+            # tests for surface families
+            if 'Surface' in family_name:
```

Comment:
```suggestion
            if 'surface' in family_name.lower():
```

Possibly make this a `lower` to avoid missing case sensitivity.